app/helpers/mono_lesson_helper.rb

```diff
@@ -0,0 +1,11 @@
+module MonoLessonHelper
+
+  def render_if_mono_lesson(options)
```

Comment:
Same comment as in https://github.com/mumuki/mumuki-domain/pull/170#issuecomment-745482737. 

It should be `Monolesson`